New version: DuckStudio.FufuDevTools version 2025.02.03.2358

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #221681

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Additional changes

- Squash 4 architecture entrance into a single `neutral` arch.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/223485)

Dragon1573 committed:
New version: DuckStudio.FufuDevTools version 2025.02.03.2358

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Luna-Grace commented on manifests/d/DuckStudio/FufuDevTools/2025.02.03.2358/DuckStudio.FufuDevTools.locale.zh-CN.yaml:
```diff
@@ -0,0 +1,43 @@
+PackageName: 芙芙工具箱开发工具包
+PackageUrl: https://github.com/DuckDuckStudio/Fufu_Dev_Tools
+License: 芙芙工具箱许可文件
+LicenseUrl: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/blob/HEAD/LICENSE
```
```suggestion
LicenseUrl: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/blob/master/LICENSE
```

Luna-Grace commented on manifests/d/DuckStudio/FufuDevTools/2025.02.03.2358/DuckStudio.FufuDevTools.locale.zh-CN.yaml:
```diff
@@ -0,0 +1,43 @@
+- 校对
+- 检查
+- 芙芙工具箱
+ReleaseNotes: |-
+  [警告] 此版本领先于 lite 版中附带的开发工具。
+
+  - fix: 修复部分功能出现找不到配置文件/相关值的问题 (#34) by @DuckDuckStudio
+
+  [提示] 这个版本也是官方的，可以放心下载使用。
+
+  Full Changelog: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/compare/2025.01.02.2300...2025.02.03.2358
```
```suggestion
ReleaseNotes: |-
  - fix: 修复部分功能出现找不到配置文件/相关值的问题 (#34) by @DuckDuckStudio
  Full Changelog: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/compare/2025.01.02.2300...2025.02.03.2358
ReleaseNotesUrl: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/releases/tag/2025.02.03.2358
```

Luna-Grace reviewed (approved):
Other than a few detail issues, it looks good to me!
CC @DuckDuckStudio

Dragon1573 committed:
Apply suggestions from code review

- Switch branch of the license, to always keep syncing with the base branch
- Reduce contents of "Release Notes"

Co-authored-by: Luna Grace <106975825+Luna-Grace@users.noreply.github.com>
